bin/ci

```diff
@@ -2,4 +2,4 @@
 
 source $(dirname $0)/config
 
-./sbt -Dsbt.log.noformat=true "; clean; stats; test-only -- exclude integration -- console junitxml; publish;  echo-version"
```

Comment:
Might have to ask @markhibberd but I believe the integration tag was here because those IAM/Roles test run for 30+ mins hence the separate saws/saws.integration builds.